docs/test/Runtime 0.1.ipynb

```diff
@@ -0,0 +1,562 @@
+    "    auth=\"cloud\", \n",
+    "    token=\"xyz\", \n",
+    "    url=\"https://cloud.ibm.com\",\n",
+    "    instance=\"<crn:v1:bluemix:public:quantum-computing:us-east:a/...> or <Quantum Services-bg>\" # full CRN or service instance name\n",
+    ")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "e1daafd6",
+   "metadata": {},
+   "source": [
+    "### Load Cloud Account from Saved Account"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "5ee3ac49",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Load default cloud account\n",
+    "service = IBMRuntimeService()"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "286dcf5a",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Load default cloud account\n",
+    "service = IBMRuntimeService(auth=\"cloud\")"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "25a7923c",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Load cloud account with name\n",
+    "service = IBMRuntimeService(name=\"my-cloud-account\")"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "6e9a4280",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Load cloud account with full CRN or service instance name\n",
+    "service = IBMRuntimeService(instance=\"crn:v1:bluemix:public:quantum-computing:us-east:a/...\") # full CRN\n",
+    "service = IBMRuntimeService(instance=\"Quantum Services-bg\") # service instance name"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "a59f2c90",
+   "metadata": {},
+   "source": [
+    "### Enable Cloud Account for current session without saving"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "b57fe754",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Enable cloud account with auth, token and instance parameters\n",
+    "service = IBMRuntimeService(auth=\"cloud\", token=\"xyz\", instance=\"<crn:v1:bluemix:public:quantum-computing:us-east:a/...> or <Quantum Services-bg>\") # full CRN or service instance name"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "f5006f52",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Enable cloud account with token and instance parameters\n",
+    "service = IBMRuntimeService(token=\"xyz\", instance=\"<crn:v1:bluemix:public:quantum-computing:us-east:a/...> or <Quantum Services-bg>\") # full CRN or service instance name"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "688a2f9b",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Enable cloud account with token, url and instance parameters\n",
+    "service = IBMRuntimeService(token=\"xyz\", url=\"https://cloud.ibm.com\", instance=\"<crn:v1:bluemix:public:quantum-computing:us-east:a/...> or <Quantum Services-bg>\") # full CRN or service instance name"
```

Comment:
Same as https://github.com/Qiskit/qiskit-ibm-runtime/pull/58/files#r777471342 

Current implementation yields `'auth' is required if 'token', or 'url' is specified but 'name' is not.`

Reply:
Fixed!